stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.mlir

```diff
@@ -178,6 +178,18 @@ func.func @attr_dict() attributes {stablehlo.attr = {attr1 = 1 : i32, attr2 = 2
   return
 }
 
+// CHECK-LABEL: "attr_custom_call_api_version_typed_ffi"
+// CHECK-NEXT: (%[[ARG0:.*]]: {{.*}})
+func.func @attr_custom_call_api_version_typed_ffi(%arg0: tensor<f32>) -> tensor<f32> {
```

Comment:
PTAL at [vhlo_checklist.md](https://github.com/openxla/stablehlo/blob/main/docs/vhlo_checklist.md), we're missing a `1.3.0` test file. We're missing a few pieces, I've covered most of these in other comments, but we also need step 3 and 4, there should be a 1.3.0 compatibility test.

Reply:
Thank you, I incorrectly assumes, since we are not creating new version of the op, we don't need these changes. 

I will update the PR.

Reply:
completed remaining tasks from the checklist.

Reply:
>  I incorrectly assumed, since we are not creating new version of the op, we don't need new version changes

Think of versioning more from the perspective of compatibility -- are we introducing something that we can't deserialize on a 1mo+ old release of StableHLO? Probably needs a new version.